.github/workflows/test.yml

```diff
@@ -8,9 +8,12 @@ on:
 jobs:
   build-and-test:
     runs-on: ubuntu-latest
+
     strategy:
       matrix:
         node: ['10', '12', '14', '16']
+      max-parallel: 1
```

Comment:
this will slow down CI but reduce the amount of "collisions" we may see due to the same tests running on the same endpoints in parallel. Considering we don't run CI on these projects very frequently, I feel the stability is worth the extra 2 minutes until we change up the test suite to not hit the real API